bin/fa5-upgrade.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+echo "Please enter your FontAwesome5 npm token:"
+
+read fa5_token
+
+echo "Setting up npm config"
+
+npm config set "@fortawesome:registry" https://npm.fontawesome.com/
+npm config set "//npm.fontawesome.com/:_authToken" $fa5_token
```

Comment:
I'm assuming that this might already have been done before. Maybe we should check whether the setting already exists and not ask for token in that case.

Reply:
I'm not sure how to implement this, I can read the first variable and only set it to a new value if it is undefined. However, the second one is secret and I can't read anything from checking ```npm config get "//npm.fontawesome.com/:_authToken"```. Is there a way to extract the value? The documentation doesn't seem to mention anything about it or the error thrown ("--- sekretz ---").